### PR TITLE
bigtable-backup-tool: Improvements

### PR DESCRIPTION
**Pull request by sandlis**
**What this PR does / why we need it**:
Added some metrics for monitoring of backup jobs
Added a command to delete out of range backups otherwise, there would be more backups than expected and alerts would start firing for unexpected backups

### CODE REVIEW

File: tools/bigtable-backup/bigtable-backup.py [hunk 1]
```diff
@@ -86,12 +73,62 @@ def ensure_backups(args):
...
+    popen = subprocess.Popen(['bigtable-backup', 'list-backups', '-ojson', '--backup-path', backup_path],
+                             stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    popen.wait()
+
+    return json.loads(popen.stdout.readline())
+
+def set_ensure_backups_specific_metrics(args, num_backups_deleted, newest_table_number, active_table_number):
+    # ensure-backups job specific metrics
+    bigtable_backup_job_tables_backed_up = Gauge('bigtable_backup_job_tables_backed_up', 'Number of backups for active and inactive tables found during last run', ['kind'], registry=registry)
+    bigtable_backup_job_backups = Gauge('bigtable_backup_job_backups', 'Sum of number of backups for all active and inactive tables found during last run', ['kind'], registry=registry)
```
Comment (gouthamve) [inline tools/bigtable-backup/bigtable-backup.py]: Number of backups for all active and inactive tables.

File: tools/bigtable-backup/bigtable-backup.py [hunk 2]
```diff
@@ -86,12 +73,62 @@ def ensure_backups(args):
...
+    bigtable_backup_job_backups_deleted = Gauge('bigtable_backup_job_backups_deleted', 'Number of backups deleted during last run', registry=registry)
+    bigtable_backup_job_expected_backups = Gauge('bigtable_backup_job_expected_backups', 'Number of backups expected for active and inactive tables', ['kind'], registry=registry)
```
Comment (gouthamve) [inline tools/bigtable-backup/bigtable-backup.py]: I see the issue being that this script _expects_ itself to be run every day. This'll fail for active table if it's being run ever other day for example.

Comment (gouthamve) [reply tools/bigtable-backup/bigtable-backup.py]: What do we get by knowing the _expected_ backups for active table?

Further, I think it's better to expose the _last_ backup time for the active table. That way will be able to an alert like active_backup is less than 36hrs old, etc.

Comment (sandlis) [reply tools/bigtable-backup/bigtable-backup.py]: right, the script should not know how frequently it is going to run.
removing expected number of backups for active table and adding timestamp of last last backup of active table.

**Pull request merged**